server/src-lib/Hasura/GraphQL/Resolve/Types.hs

```diff
@@ -262,12 +261,47 @@ data InputFunctionArgument
   | IFAUnknown !FunctionArgItem -- ^ Unknown value, need to be parsed
   deriving (Show, Eq)
 
+{- Note [Relay Node Id]
```

Comment:
@rakeshkky Let's add version as the `0`th element of the array. The reason being that if these GUIDs end up in URLs or if apps depend on them to not change as their schema evolves, an encoding which uses `(schema_name, table_name)` is not good enough (but it is the best we have currently).

 In future we may want to provide some sort of an identifier that can be attached to each table and this identifier is used to encode GUIDs for that table rather than (schema_name, table_name). So to accommodate for such changes, we'll need a version field in the ID.